meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,241 @@
+# v1.3.0 release changelogs
+
+Meilisearch v1.3.0 introduces Meilisearch as a vector store, it introduces visible ranking score details and defining a field to search on at search time.  It also now includes the ability to search within facet values and facet values that are sortable.
+
+<!-- The following lines should ONLY be put in the official release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+Some SDKs might not include all new features—consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and improvements 🔥
+
+## Vector Store - Experimental
+The release of v1.3.0 enables Meilisearch to serve as a vector store. This means Meilisearch now accepts vectors in documents, stores them, and lets users retrieve the nearest documents using the search endpoint.
+
+This experimental feature can be enabled at runtime using v1.3.0's new `/experimental-features` endpoint.
+```bash
+curl \
+  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json'  \
+--data-binary '{
+    "vectorSearch": true
+  }'
+
+```
+
+### Sending Vectorized Documents
+
+For the first iteration of this feature you must compute the vectors on your side before sending to Meilisearch. Vectors should be sent with the document itself using the `_vectors` field. Multiple vectors can be sent for a single document.
+
+⚠️ Note the size of the vectors must be the same across all of your documents. If the sizes are inconsistent Meilisearch will show an error in your tasks.
+
+```bash
+curl -X POST -H 'content-type: application/json' \
+'localhost:7700/indexes/myvectors/documents' \
+--data-binary '[
+    { "id": 0, "_vectors": [0, 0.8, -0.2], "text": "Hello World" },
+    { "id": 1, "_vectors": [1, -0.2, 0], "text": "Hello Doggy" },
+    { "id": 2, "_vectors": [[0.5, 3, 1], [-0.2, 4, 6]], "text": "Hello People" }
+]'
+```
+
+### Query Meilisearch using Vectors
+
+You can query the nearest documents using the default `/search` route by sending the vector of your query via the `vector` field. Again, for the first iteration you must compute the vector query on your side before requesting from Meilisearch.
+
+```bash
+curl -X POST -H 'content-type: application/json' \
+'localhost:7700/indexes/myvectors/search' \
+--data-binary '{ "vector": [0, 1, 2] }'
+```
+
+### Similarity score
+As part of v1.3.0 Meilisearch will return a similarity score as part of the documents retrieved. This score will be between 0 and 1 and viewed under the `_semanticSimilarity` property. The higher the ranking score, the more similar the document.
```

Comment:
Ho! The score field has been renamed to `_semanticScore` 😊
You can find more info in [this discussion](https://github.com/meilisearch/product/discussions/677).

Reply:
Can we add that the score is a dot product and that's it for now?

Reply:
Thanks! Just to clarify do you mean for me to say 'The score is a dot product viewed under the _semanticScore property' (and remove the higher the score comment?)

Reply:
The examples show it like '_semanticScore": 0.6754 ' if that's what you mean?

Reply:
No, I wanted to specify that the score is in fact a dot product. Maybe you can add that somewhere? :)

Reply:
thats updated now